docs/nodes.md

```diff
@@ -52,7 +52,7 @@ change your inventory to:
 
 ## 2) Upgrade the cluster
 
-run `cluster-upgrade.yml` or `cluster.yml`. Now you are good to go on with the removal.
+run `upgrade.yml` or `cluster.yml`. Now you are good to go on with the removal.
```

Comment:
upgrade.yml exists under roles/network_plugin/calico/tasks/ only.
I guess `upgrade-cluster.yml` under the root directory instead.

Reply:
🤦